distributions/standalone/src/main/resources/apps/IGNORE

```diff
@@ -0,0 +1,3 @@
+Git does not allow versioning of an empty folder so we add this file here to create the folder in git.
```

Comment:
This file doesn't go into the distro. Is the same as we do in domains. We put it here so git can track the folder but then we ignore it in assembly.xml